bundles/org.openhab.ui/web/src/components/config/controls/item-picker.vue

```diff
@@ -115,7 +116,11 @@ export default {
     this.smartSelectParams.closeOnSelect = !(this.multiple)
     if (this.setValueText) this.smartSelectParams.setValueText = this.setValueText
     if (!this.items || !this.items.length) {
-      this.$oh.api.get('/rest/items?staticDataOnly=true').then((items) => {
+      let url = '/rest/items?staticDataOnly=true'
+      if (this.filterTag) {
+        url = url + '&tags=' + this.filterTag.join('%2C')
```

Comment:
Doesn't JS have some proper URL/URI class/object that can build URLs and handle URL encoding etc.?